### PR TITLE
{numlib}[GCC/8.3.0-2.32] OpenBLAS v0.3.6 (in preparation for 2019b toolchains)

### PR DESCRIPTION
**Pull request by edmondac**

### CODE REVIEW

**Comment by edmondac** [comment]
See https://github.com/easybuilders/easybuild-easyblocks/pull/1765

**Comment by edmondac** [comment]
@bartoldeman https://github.com/xianyi/OpenBLAS/releases says "the AVX512 DGEMM kernel has been disabled again due to unsolved problems" - is that enough?

**Comment by bartoldeman** [comment]
No. Some more things needed disabling too, and that patch was committed to OpenBLAS git a week after the 0.3.6 release.

**Comment by edmondac** [comment]
Ah - sorry. I missed that you'd supplied the 0.3.6 patch... now included.

**Comment by edmondac** [comment]
@bartoldeman does that look ok? And also do you know how to poke Travis? It looks like it blew up to do with apt...

File: easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.6-GCC-8.3.0-2.32.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+source_urls = [
+    # order matters, trying to download the large.tgz/timing.tgz LAPACK tarballs from GitHub causes trouble
+    'http://www.netlib.org/lapack/timing/',
+    'https://github.com/xianyi/OpenBLAS/archive/',
+]
+sources = ['v%(version)s.tar.gz']
+patches = [
+    ('large.tgz', '.'),
+    ('timing.tgz', '.'),
+]
```
Comment (bartoldeman) [inline easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.6-GCC-8.3.0-2.32.eb]: the patch is still missing here?

Comment (edmondac) [reply easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.6-GCC-8.3.0-2.32.eb]: Doh. Added the wrong file. Fixed now.

**Comment by boegel** [comment]
Test reports should come in soon (any chance for a test report from a POWER box for this one @edmondac?).

W.r.t. re-triggering Travis: the "supported" way (according to the Travis support team) is to simply close & re-open the PR.
Some people (incl. your truly) can manually trigger individual tests if they fail because of a fluke, but usually my @boegelbot will beat you to it: it checks Travis logs for failing tests, and can detect the most common flukes; if it sees one, it'll retrigger that Travis job for us fully automatically!

That's some true AI for you... ;)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2617.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/389dc7ca2dd0f45ed71f09d4b35ca834 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2123.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d1c7c6734ad36ef6f7d3534a9414e270 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (1 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/e834e02ad7222c0c2945581e1ad134f6 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/00c29d36c955faae9dfc9cf99026c47a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @edmondac!

**Comment by edmondac** [comment]
Test report by @bear-software
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0305u15a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/77ed0106697ddc0f931d220e189010cf for a full test report.

**Pull request merged**